host_guest/cyclodextrin_derivatives/README.md

```diff
@@ -20,7 +20,7 @@ We plan to separately analyze submissions predicting absolute binding free energ
 
 ## Challenge timing details (preliminary)
 
-Dr. Kellett has finished the ITC data collection on guest binding and [full experimental details are available](../../host_guest_description.md). Participants may begin their calculations (though we will be preparing additional structures/inputs for those who would like additional file formats) and submissions will be allowed as soon as our submission system is online. The challenge is expected to run through August, 2019, with experimental results released at that time. (The formal submission deadline will be announced soon.)
+Dr. Kellett has finished the ITC data collection on guest binding and [full experimental details are available](../../host_guest_description.md). Participants may begin their calculations (though we will be preparing additional structures/inputs for those who would like additional file formats) and submissions will be allowed as soon as our submission system is online. The challenge is Oct., 2019, with experimental results released at that time. The formal submission deadline is available on the main page for this repository.
```

Comment:
I think we got all the input files up? (ie pdb, mol2, sdf?)

Reply:
I just double checked and there are PDB, mol2 and SDF files for all the guests and hosts for the CD challenge

Reply:
Ah, so you're referring to :
> (though we will be preparing additional structures/inputs for those who would like additional file formats)

In other words, I could remove this... 

I'll update to fix this.

Reply:
Yes. I got worried for a second that we wanted more input formats!